contracts/src/v0.8/workflow/dev/v2/WorkflowRegistry.sol

```diff
@@ -149,6 +149,19 @@ contract WorkflowRegistry is Ownable2StepMsgSender, ITypeAndVersion {
     bytes attributes; //         Arbitrary bytes for additional workflow details.
   }
 
+  struct WorkflowMetadataView {
```

Comment:
will leave it to you to determine if we want to pack this struct:
https://github.com/RensR/Spack